ts/store/reducers/entities/messages/index.ts

```diff
@@ -6,32 +6,32 @@ import { combineReducers } from "redux";
 
 import { Action } from "../../../actions/types";
 
-import { MessageState } from "./messagesById";
-import { MessageStatus } from "./messagesStatus";
 import allPaginatedReducer, { AllPaginated } from "./allPaginated";
 import detailsByIdReducer, { DetailsById } from "./detailsById";
 import paginatedByIdReducer, { PaginatedById } from "./paginatedById";
 import { thirdPartyByIdReducer, ThirdPartyById } from "./thirdPartyById";
 import { Downloads, downloadsReducer } from "./downloads";
+import {
+  ThirdPartyMessagePreconditionById,
+  thirdPartyMessagePreconditionByIdReducer
+} from "./thirdPartyMessagePreconditionById";
 
 export type MessagesState = Readonly<{
   allPaginated: AllPaginated;
   paginatedById: PaginatedById;
   detailsById: DetailsById;
   thirdPartyById: ThirdPartyById;
   downloads: Downloads;
+  thirdPartyMessagePreconditionById: ThirdPartyMessagePreconditionById;
 }>;
 
 const reducer = combineReducers<MessagesState, Action>({
   allPaginated: allPaginatedReducer,
   paginatedById: paginatedByIdReducer,
   detailsById: detailsByIdReducer,
   thirdPartyById: thirdPartyByIdReducer,
-  downloads: downloadsReducer
+  downloads: downloadsReducer,
+  thirdPartyMessagePreconditionById: thirdPartyMessagePreconditionByIdReducer
```

Comment:
See comment on `ts/store/reducers/entities/messages/messagesById.ts`

Reply:
Removed all references to `thirdPartyMessagePreconditionById`